<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>multiMatteWin</class>
 <widget class="QMainWindow" name="multiMatteWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_13">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,2,0,0,0,0,0">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Material</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="material_listWidget">
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="selectMtr_checkBox">
          <property name="text">
           <string>Select Material</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="refresh1_pushButton">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="1" column="1">
              <widget class="QLineEdit" name="start_lineEdit"/>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="id_lineEdit"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Start </string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="increment_lineEdit"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Increment</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>current ID </string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QCheckBox" name="assignAll_checkBox">
               <property name="text">
                <string>Assign All</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="assign_puhsButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Assign</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="undo_pushButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Undo last assigned</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9" stretch="0,2,0,0,0,0,0">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>MultiMatte</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="multiMatte_listWidget"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QCheckBox" name="checkBox_5">
            <property name="text">
             <string>Select MultiMatte</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_4">
            <property name="text">
             <string>Select Material List</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="refresh2_pushButton">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="1" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Prefix</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Current Name</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="selName_lineEdit"/>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="prefix_lineEdit"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="name_lineEdit"/>
             </item>
             <item row="3" column="0">
              <widget class="QCheckBox" name="createAll_checkBox">
               <property name="text">
                <string>Create All</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="createMultiMatte_pushButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Create MultiMatte</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reAssign_pushButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>ReAssign MultiMatte</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
